admin-data-explorer-play-server/src/test/scala/controllers/AdminControllerSpec.scala

```diff
@@ -200,11 +459,44 @@ class AdminControllerSpec extends PlayPostgresSpec {
       samePassword must be(true)
     }
 
+    "return table rows for all tables" in withApiClient { client =>
+      val tables = List(serialSettings, bigSerialSettings)
+      val numOfCreatedRows = 3
+      for (table <- tables) {
+        Range.apply(0, numOfCreatedRows).foreach { i =>
```

Comment:
There are shortcuts to this, `(0 to numOfCreatedRows).foreach { ... }` and `(0 until numOfCreatedRows).foreach { ... }`